appcenter-data/Data.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const ReactNative = require('react-native');
+
+const { AppCenterReactNativeData } = ReactNative.NativeModules;
+
+const Data = {
```

Comment:
We are adding a new native module to RN SDK, it would be good to add Jest unit test to for **Data.js**. Since RN SDK is merely a "thin" wrapper on native (Android/iOS) SDKs, we don't need to test the behavior of native SDKs' APIs. Instead, the focus should be on the bridge code calls the right native APIs. 

Specifically for the scope of this PR,
- AppCenterReactNativeDataModule.java invokes the right Java APIs of native Android.

If you are not sure, please see an example of `appcenter-analytics` at https://github.com/microsoft/appcenter-sdk-react-native/blob/develop/appcenter-analytics/__tests__/Analytics.js.

Reply:
Sounds good. I'll create a separate work item to add tests.